css/index.css

```diff
@@ -101,17 +122,56 @@ p {
     justify-content: space-evenly;
 }
 
+.boxes .box:nth-of-type(1) {background: teal;}
+.boxes .box:nth-of-type(2) {background: gold;}
+.boxes .box:nth-of-type(3) {background: cadetblue;}
+.boxes .box:nth-of-type(4) {background: coral;}
+.boxes .box:nth-of-type(5) {background: crimson;}
+.boxes .box:nth-of-type(6) {background: forestgreen;}
+.boxes .box:nth-of-type(7) {background: darkorchid;}
+.boxes .box:nth-of-type(8) {background: hotpink;}
+.boxes .box:nth-of-type(9) {background: indigo;}
+.boxes .box:nth-of-type(10) {background: dodgerblue;}
```

Comment:
Another approach that could have been used instead of ```:nth-of-type``` would have been to assign a second class name such as ```.box1``` for each